### PR TITLE
Adding Power support(ppc64le) with ci and testing to the project for architecture independent

### PR DESCRIPTION
**Pull request by asellappen**
I am working for IBM to port cpu arch ppc64le for open sources.

This PR adds CI support for the IBM Power Little Endian (ppc64le) architecture. The idea is to ensure that the builds on this architecture are continuously tested along with the Intel builds (amd64) as this is part of the ubuntu distro 
on that architecture as well and detecting (and fixing) any issues or failures early would help to ensure that we are always up to date.

This helps us simplify testing later when distributions are re-building and re-releasing,We typically build applications for customers and 
ISVs, and while we don't use this package directly,we do count on all of the packages in debian/ubuntu to build other packages. So we more likely have this as a second or 
third level dependency and couldn't tell you explicitly which features we use or our usage model,For more info tag @gerrith3.

Please help to verify and merge.

### CODE REVIEW

File: .travis.yml
Comment (ThomasWaldmann) [inline .travis.yml:6]: dropping 3.4 / ubuntu trusty testing for linux is not in scope of this PR.

if 3.4 / trusty does not work for ppc, just leave it away there.

**Review by ThomasWaldmann — CHANGES_REQUESTED** [review]
thanks for the PR, just found one little nitpick.

we still announce py34 support via pypi metadata, so you must not remove testing for that.

**Comment by asellappen** [comment]
thanks for looking at the PR ,I have added 3.4 for amd64 now ,Please check and merge.

Pls not i have not included 3.4 for ppc64le as the version is not available to download from Downloading archive: https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/18.04/ppc64le/python-3.4.tar.bz2.

Thanks again

**Comment by ThomasWaldmann** [comment]
Looks good, I'll wait until travis finished, then it's merge time!

**Comment by gerrith3** [comment]
Hey @ThomasWaldmann it looks like you are still registered with travis-ci.ORG - you might want to move to travis-ci.COM.  Should be equivalent functionality for the most part but builds on .ORG are slowed down to encourage people to move to .COM.  .ORG will be shut down as they merge its previous functionality with the .COM site.

**Comment by gerrith3** [comment]
BTW if you need additional credits on Travis, they are free for open source maintainers/projects - but you have to request via support.travis-ci.com, ideally once you have migrated there.

**Comment by ThomasWaldmann** [comment]
strange. why do we suddenly have a py34 failure? it does not seem like related to this PR.

**Comment by gerrith3** [comment]
@ThomasWaldmann yeah - I saw that and it doesn't seem related and it is on Intel which shouldn't have been changed?  I'm wondering if some travis environment variable was defaulted in the change and that's the issue?  @asellappen is in India and can look at it during his day to see if it is a .travis.yml issue.

**Comment by gerrith3** [comment]
@ThomasWaldmann actually, reviewing the changes, I don't see how it could have possibly affected the py34 build.  I don't know if there are differences in the back ends for Intel - I do know that our .org environment could be a little out of date compared to the ppc64le environment on travis-ci.com, but that isn't the issue here at all.

**Comment by asellappen** [comment]
not changed anything in travis environment ,just add power on support in yml script,It's bit strange .

**Comment by ThomasWaldmann** [comment]
looks like introduced before your PR...

**Comment by ThomasWaldmann** [comment]
Guess this will stop working when we do #47. :-(

**Pull request merged**
